src/_sass/pages/_go-contactless.scss

```diff
@@ -38,14 +38,14 @@ $gc-blue: rgb(235, 248, 252);
 }
 
 .go-contactless-benefit-card,
-.go-contactless-agency {
+.agency-card {
```

Comment:
Really nitpicky but these aren't technically Cards. A Card has is an unit that has an image and text combined (https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Card). This repo does use Bootstrap, which has its own thing called Cards, so I don't think I would use the naming `card` here because it might make it seem like we are using Bootstrap Card stuff here. 

<img width="1512" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/4d5f5c85-2f77-41e9-9899-efa7790d6aa3">

Reply:
Got it. I renamed it to just `agency` in https://github.com/cal-itp/mobility-marketplace/pull/491/commits/02bd720de6ffc057d17c16753328fab862d23b59